chapters/synchronous.tex

```diff
@@ -726,6 +726,22 @@ \subsection{Sub-Clock Conversion Operators}\label{sub-clock-conversion-operators
 Clock y1 = shiftSample(u, 2);       // ticks: 2.0, 3.0, $\ldots$
 Clock y2 = shiftSample(u, 2, 3);    // error (resolution must be 1)
 \end{lstlisting}
+Additional example showing the full form:
+\begin{lstlisting}[language=modelica]
+  Integer intervalCnt(start=2);
+  Integer cnt(start=0);
+  Clock u = Clock(intervalCnt,1);
+  Clock s1 = shiftSample(u, 3, 2);
+equation
+   when u then
+     cnt = previous(cnt) + 1;
+     intervalCnt = if (cnt>=2) then 1 else previous(intervalCnt);
+   end when;
+\end{lstlisting}
+Here \lstinline!u! ticks at \lstinline!0, 2, 3, 4, 5, 6!.
+First you \lstinline!superSample! to split each sampling interval in two equal parts leading to the ticks \lstinline!0.0, 1.0, 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5.0, 5.5, 6.0!.
+Then the simple \lstinline!shiftSample! removes the first three ticks giving \lstinline!2.5, 3.0, 3.5, 4.0, 4.5, 5.0, 5.5, 6.0!.
+And finally every other point is removed by \lstinline!subSample!, and \lstinline!s1! ticks at \lstinline!2.5, 3.5, 4.5, 5.5!.
```

Comment:
Can't we have all these time values as normal numbers instead of inline code?  I think it would both make sense semantically and avoid possible typesetting issues with line breaking of inline code:
```suggestion
Here \lstinline!u! ticks at 0, 2, 3, 4, 5, 6.
First you \lstinline!superSample! to split each sampling interval in two equal parts leading to the ticks 0.0, 1.0, 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5.0, 5.5, 6.0.
Then the simple \lstinline!shiftSample! removes the first three ticks giving 2.5, 3.0, 3.5, 4.0, 4.5, 5.0, 5.5, 6.0.
And finally every other point is removed by \lstinline!subSample!, and \lstinline!s1! ticks at 2.5, 3.5, 4.5, 5.5.
```

Reply:
Sure.